### ISSUE TITLE
Fetaure request: Mutliple accounts for accessing the server web interface

### ISSUE THREAD

**Issue by NewLunarFire** [colloqi/pisignage-server#44]
We needed several different accounts for accessing the server web interface. PiSignage supports only one, so we used another way to do this. But it would be nice to support multiple accounts in PiSignage.

The work-around: We used our nginx reverse proxy to control access with basic auth. On success, we send the 'Authorization' header with the piSignage credentials to pisignage-server.

**Comment by codeagencybe** [comment]
Any update on this? Is this already supported?

**Comment by NewLunarFire** [comment]
No. I just verified and this hasn't been changed since. We're still using our nginx proxy and it's working fine.

**Comment by badohk** [comment]
Hello did you manage to have several different users on your opensource server? If so, how could I explain? Please thank you.

**Comment by hitesh83** [comment]
@badohk there is no way to have a different users with their own playlist and players. @NewLunarFire is using reverse proxy and nginx to send different users of nginx to same single user interface of Pisignage. Every one sees same list of players with same assets files.
For multiple user please go to pisignage.com hosted solution.

**Comment by bigpappa456** [comment]
Could you please tell me  to do this, create multiple users with reverse proxy, it's good for me that they go to same user interface but i wish to create more users that can login on the open source server
